scripts/check-go-version.sh

```diff
@@ -8,13 +8,17 @@ while getopts "uvd" opt; do
     case $opt in
         u) update=true ;;
         v) verbose=true ;;
-        x) set -x ;;
         *) echo "Usage: $0 [-u] [-v] [-d]" >&2
            exit 1
            ;;
     esac
 done
 
+# Debugging option (-x) used separately
```

Comment:
I don't follow. The linter warning was `This case is not specified by getopts.`, so we just need to add it there
```
while getopts "uvdx" opt; do
```